WordPress/src/main/res/layout/restore_complete_fragment.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <androidx.recyclerview.widget.RecyclerView
+        android:id="@+id/recycler_view"
+        android:layout_width="match_parent"
```

Comment:
Nitpick 🔍:`MATCH_PARENT` is not recommended for widgets contained in a `ConstraintLayout`.

Reply:
I'll address this during consolidation, as all fragments restore/backup will use the same fragment file. Thanks.